packages/api/src/schema/db-schema.yaml

```diff
@@ -1385,3 +1385,15 @@ components:
                       type: string
                     timestamp:
                       type: integer
+    new-asset-from-url-payload:
+      properties:
+        url:
+          type: string
+          description: URL of the asset to be imported
```

Comment:
This doesn't need to be repeated here, the db-schema is deep merged!

Reply:
Cool!, we can remove the reference of `url` from here?